spec/chewy/fields/base_spec.rb

```diff
@@ -400,6 +400,313 @@
       end
     end
 
+    context 'ignore_blank option for field method', :orm do
+      before do
+        stub_model(:location)
+        stub_model(:city)
+        stub_model(:country)
+
+        City.belongs_to :country
+        Location.belongs_to :city
+        City.has_one :location, -> { order :id }
+        Country.has_many :cities, -> { order :id }
+      end
+
+      context 'text fields type with and without ignore_blank option' do
+        before do
+          stub_index(:countries) do
+            define_type Country do
+              field :id
+              field :cities do
+                field :id
+                field :name
+                field :historical_name, ignore_blank: false
+                field :description, ignore_blank: true
+              end
+            end
+          end
+        end
+
+        let(:country_with_cities) do
+          cities = [
+            City.create!(id: 1, name: '', historical_name: '', description: ''),
+            City.create!(id: 2, name: '', historical_name: '', description: '')
+          ]
+
+          Country.create!(id: 1, cities: cities)
+        end
+
+        specify do
+          expect(CountriesIndex::Country.root.compose(country_with_cities)).to eq(
+            'id' => 1, 'cities' => [
+              {'id' => 1, 'name' => '', 'historical_name' => ''},
+              {'id' => 2, 'name' => '', 'historical_name' => ''}
+            ]
+          )
+        end
+      end
+
+      context 'fields with ignore_blank: true flag with .blank? method' do
+        before do
+          stub_index(:countries) do
+            define_type Country do
+              field :id
+              field :cities do
+                field :id
+                field :name, ignore_blank: true
+                field :historical_name, ignore_blank: true
+                field :description, ignore_blank: true
+              end
+            end
+          end
+        end
+
+        let(:country_with_cities) do
+          cities = [
+            City.create!(id: 1, name: '', historical_name: '', description: nil)
+          ]
+
+          Country.create!(id: 1, cities: cities)
+        end
+
+        specify do
+          expect(CountriesIndex::Country.root.compose(country_with_cities)).to eq(
+            'id' => 1, 'cities' => [
+              {'id' => 1}
+            ]
+          )
+        end
+      end
+
```

Comment:
Does this context add some cases, not handled by the previous one? I couldn't find any.
```suggestion
```